Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -194,8 +194,8 @@ export default class CovidCardView extends Vue {
     }
 
     private setAddress(
-        defaultAddress: Address | undefined,
-        backupAddress: Address | undefined
+        defaultAddress: Address | null,
+        backupAddress: Address | null
```

Comment:
looking at the lines 200-206 will merge with the existing address. Should it not replace them if not null?

Reply:
This function sets the address to either a clone of defaultAddress (if it's not null), a clone of backupAddress (if defaultAddress is null and backupAddress isn't), or an empty object (if both are null).

Reply:
Looks like JS does not copy the values of the object but the pointer. With that, we should be using the spread operator (as in lines 200-206) for coping values